gfx_window.h

```diff
@@ -135,7 +134,7 @@ class GfxWindowInternal
     {
         if(window_)
             DestroyWindow(window_);
-        if(is_imgui_initialized_)
+        if(ImGui_ImplWin32_GetBackendData() != nullptr)
```

Comment:
There was an issue with ```is_imgui_initialized_ getting``` inside gfx_window getting out of sync with gfx_imgui. ```ImGui_ImplWin32_GetBackendData()``` prevents this as it is always null if imgui has not been initialised. Using this directly prevents a crash that occurs occasionally when gfxImgui*** functions are called resulting in a imgui shutdown that gfx_windows isnt aware of.

Reply:
Ah, that does sounds like a good change indeed, maybe we can get rid of that `is_imgui_initialized_` altogether then?

Reply:
I did ;) (line 81)